### PR TITLE
Added program to reverse words of string in c++.

### PR DESCRIPTION
**Pull request by maanpatel2492**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES

### Description

https://github.com/TesseractCoding/NeoAlgo/issues/2122#issuecomment-791581760

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Pull Request has a program to display words in the reverse order of the string.It is in C++.

### CODE REVIEW

**Comment by HarshCasper** [comment]
Add the Issue Link in your PR Description. The PR would be reviewed after the Coding Period of GSSoC starts. Thanks!

**Comment by Simi06** [comment]
Can I please have this assigned to me?

File: Rev_str.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    cout << tmp[0] << endl;
+}
+
+int main()
+{
+    string str;
+    getline(cin, str);
+    rev_words(str);
+    return 0;
+}
```
Comment (Vishakha-sta) [inline Rev_str.cpp]: /*
test case 1:
input:- 

Hello World!

output:-

World! Hello

test case 2:
input:- 

Hello     .        World!

output:-

World!        .     Hello

*/

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Few changes: 

- Add a beginner friendly pitch at the top of the Code.
- Add Time and Space Complexity at the end of the Code
- Add sample I/O at the end of the Code
- Add comments where possible

**Comment by maanpatel2492** [comment]
> Few changes:
> 
> * Add a beginner friendly pitch at the top of the Code.
> * Add Time and Space Complexity at the end of the Code
> * Add sample I/O at the end of the Code
> * Add comments where possible

Sure.

**Comment by atarax665** [comment]
@maanpatel2492  Are you still working on this?

**Comment by maanpatel2492** [comment]
> @maanpatel2492 Are you still working on this?

Yes,
I have to make above changes and create pull request again right?

**Comment by HarshCasper** [comment]
> > @maanpatel2492 Are you still working on this?
> 
> Yes,
> I have to make above changes and create pull request again right?

Make the changes in the same PR

**Comment by ankitaggarwal23** [comment]
@maanpatel2492 All contributions done by participants from 8 March in GSSoC'21 Projects will be counted. Thanks, Closing this as of now.

**Comment by maanpatel2492** [comment]
> @maanpatel2492 All contributions done by participants from 8 March in GSSoC'21 Projects will be counted. Thanks, Closing this as of now.


There is invalid tag.Does I made any mistake.

**Comment by ankitaggarwal23** [comment]
Code plagiarized from [here](https://www.geeksforgeeks.org/reverse-words-in-a-given-string/)

**Pull request closed without merging**